### ISSUE TITLE
Wan connection new packages not connecting to the Internet

### ISSUE THREAD

**Issue by m0lz** [negan07/ancistrus#90]
Hi Negan07

I have a problem with V1.0.1.84_1.0.1 build #916, it will not connect to the Internet.

If I roll back to V1.0.1.82_1.0.1 build #875 there are no problems.

I have tried flashing the D7000 twice now with the new build, each time also reloading my backup settings, and in both cases the router will not connect to the internet. I flashed build 916, then reflashed 875 and the internet connected. Again flashed 916, no internet again, and then reflashed 875 and the Internet is now working again.

I do not have a corrupt download, and the router settings all seem fine after the flash of 916, it just refuses to connect to the internet and I have an Orange LED for internet connection with 916 installed.

Let me know what you need to look into this 👍

**Comment by Linden1** [comment]
I can confirm this issue. PPPoE over ATM was fine when I tested. Now it's
been integrated I can't connect. Wound back to #875 for now.

Linden Gill

**Comment by negan07** [comment]
wan connection have been updated

which package is involved ? (more than one possible)

```
ppp
rp-pppoe
brctl
```

**Comment by Linden1** [comment]
Going back to issue #88 this is the same ipk I installed on 875 and confirmed was working on PPPoE over WAN(ATM). I'll give it a try on 916

**Comment by Linden1** [comment]
Confirming the package in wan_orig_0.0.0_armD7000.zip fixes this issue.
Flash 916, install the package negan07 has linked here and Reboot.

**Comment by m0lz** [comment]
> wan connection have been updated on #916
> 
> this package should revert wan to original (unzip & manual install from package manager): installing it there's no need to reflash [wan_orig_0.0.0_armD7000.zip](https://github.com/negan07/ancistrus/files/8596708/wan_orig_0.0.0_armD7000.zip)
> 

Package manager?
Sorry but I only know how to update the firmware with an .img, that .ipk file is in the realms of voodoo magic to me :)

**Comment by Linden1** [comment]
![IMG20220430122254.jpg](https://user-images.githubusercontent.com/73253891/166103524-6c778de6-bd2f-4231-aea4-15e640fa3174.jpg)

**Comment by m0lz** [comment]
Ah I see, so I need to install once again the 1.0.1.84, then use the package manager to overwrite that package and test?

**Comment by Linden1** [comment]
> Ah I see, so I need to install once again the 1.0.1.84, then use the package manager to overwrite that package and test?

Yes. It's that simple. Don't forget to Reboot after.

**Comment by negan07** [comment]
> Going back to issue #88 this is the same ipk I installed on 875 and confirmed was working on PPPoE over WAN(ATM). I'll give it a try on 916

so the package `ppp-2.4.9` is fully working with atm connection (adsl) ?
any test with ptm ?

please report which test have been made with which connection to isolate the faulty package

**Comment by Linden1** [comment]
PPPoE over WAN(ATM) - working 👍

**Comment by negan07** [comment]
please report which test have been made with which package and with which connection to isolate the faulty package

e.g. starting from old original wan packages or from #875 build install & reboot once per time `linux-atm, ppp, rp-pppoe, bridge-utils` (even if this last one should be involved...)

**Comment by m0lz** [comment]
I just updated the firmware from 875 to 916 again
then installed the wan orig .ipk file
rebooted

And the internet connection is working again
VDSL(PTM) - VDSL(2)
PPPoE

**Comment by negan07** [comment]
[ppp_2.4.9-001_armD7000.zip](https://github.com/negan07/ancistrus/files/8596929/ppp_2.4.9-001_armD7000.zip)

If someone wanna try, I've recompiled ppp without any further vendor patch modifications
unzip & manual install

**Comment by Linden1** [comment]
Build #875
PPPoE over WAN (ATM) > connecting fine.
install linux-atm 2.5.2 > reboot > connecting fine.
install ppp 2.4.9 > lose connection > reboot > no connection (LED orange).

output from cat /tmp/ppp_log: cat: can't open '/tmp/ppp_log': No such file or directory

Installed wan_orig package > reboot > connecting fine.

**Comment by Linden1** [comment]
ppp_2.4.9-001_armD7000 breaks it too :(

**Comment by Discovery00** [comment]
I tested 3 packages, they work fine

**Comment by Discovery00** [comment]
Please install with package manager

**Comment by Discovery00** [comment]
Install Atm first then ppp, rp-pppoe
![IMG_20220430_222158](https://user-images.githubusercontent.com/100864172/166119732-8c70e1f9-41ef-483f-be5e-68d68e2e319b.jpg)

**Comment by Discovery00** [comment]
https://github.com/negan07/ancistrus/issues/88#issuecomment-1100432170
Looks at this log file :) I installed on build 875 via package manager. Test OK

**Comment by Linden1** [comment]
> Install Atm first then ppp, rp-pppoe
> ![IMG_20220430_222158](https://user-images.githubusercontent.com/100864172/166119732-8c70e1f9-41ef-483f-be5e-68d68e2e319b.jpg)
> 

With 916 all the packages are preinstalled. Overwriting with the wan_orig only changes ppp to an earlier package. So I have them all already, except ppp

**Comment by Discovery00** [comment]
I think Bridge-utils causes this problem

**Comment by Discovery00** [comment]
Send your ppp log file on build 916 please

**Comment by Linden1** [comment]
I've even tried using startup.sh to build from D7000-V1.0.1.84_1.0.1_rescue.img and still this same issue.

**Comment by Linden1** [comment]
> Send your ppp log file on build 916 please

still getting the error
can't open '/tmp/ppp_log': No such file or directory

**Comment by negan07** [comment]
This is the only package not involved into chap auth connection stage

`brctl` simply creates bridges between interfaces
executing:
`brctl show`
or
`ip link
`or 
`ifconfig`

you should see at least `group1`

The faulty package should be `ppp-2.4.9`

From a clean firmware or from #875 installing all except `ppp-2.4.9` should result in a successful connection

**Comment by negan07** [comment]
> > Install Atm first then ppp, rp-pppoe
> > ![IMG_20220430_222158](https://user-images.githubusercontent.com/100864172/166119732-8c70e1f9-41ef-483f-be5e-68d68e2e319b.jpg)
> 
> With build #916 all the packages are preinstalled. Overwriting with the wan_orig only changes ppp to an earlier package. So I have them all already, except ppp - confirmed with package manager. This issue seems to be different to the one you had in #88

I agree that the issue related refers to a different auth connection problem, also present on original vendor `ppp-2.4.1`

**Comment by Linden1** [comment]
> This is the only package not involved into chap auth connection stage
> 
> `brctl` simply creates bridges between interfaces executing: `brctl show` or `ip link `or `ifconfig`
> 
> you should see at least `group1`
> 
> The faulty package should be `ppp-2.4.9`
> 
> From a clean firmware or from #875 installing all except `ppp-2.4.9` should result in a successful connection

It does - confirmed

**Comment by negan07** [comment]
It would be useful to investigate if the problem happens with `ptm` (vdsl) connections too
It seems that all the trials here have been done with `atm` (adsl) connection: is it right ?

**Comment by Linden1** [comment]
> It would be useful to investigate if the problem happens with `ptm` (vdsl) connections too It seems that all the trials here have been done with `atm` (adsl) connection: is it right ?

for me, yes ATM

**Comment by negan07** [comment]
[ppp_2.4.9-002_armD7000.zip](https://github.com/negan07/ancistrus/files/8597807/ppp_2.4.9-002_armD7000.zip)
I have recompiled ppp-2.4.9 with brcm patch support except for the new `libpppoe` remained unchanged (latest one is used)

In case of failure, it would be useful looking for any kind of helpful error messages apart "no connection" e.g. in the router log message events on advanced gui or http://192.168.0.1/ppp_log or /tmp/ppp_log if any

**Comment by negan07** [comment]
[ppp_2.4.9-002_armD7000.zip](https://github.com/negan07/ancistrus/files/8597807/ppp_2.4.9-002_armD7000.zip)
I have recompiled ppp-2.4.9 with brcm patch support except for the new `libpppoe` remained unchanged (latest one is used)

In case of failure, it would be useful looking for any kind of helpful error messages apart "no connection" e.g. in the router log message events on advanced gui or http://192.168.0.1/ppp_log or /tmp/ppp_log if any

**Comment by Discovery00** [comment]
http://192.168.0.1/?todo=ppp_debug

**Comment by Discovery00** [comment]
First of all, you must be enable ppp debug.

**Comment by Discovery00** [comment]
Greetings :)

**Comment by Discovery00** [comment]
Please try enabling this. then send your ppp log file

**Comment by Discovery00** [comment]
Negan, Netgear Closed PPP debug function in latest firmware. Do you know? I cant enable on Original 0.84 release.

**Comment by Linden1** [comment]
ppp_2.4.9-002_armD7000

Working for ATM on build 619 😬

**Comment by m0lz** [comment]
Just come home from night shift, I need to find out if my daughter is working remotely from home today first, if not then I will test ppp_2.4.9-002_armD7000 in a few hours on VDSL

**Comment by m0lz** [comment]
VDSL

Installed Build #916
Rebooted
Installed ppp_2.4.9-002_armD7000.zip with package manager
Rebooted

Internet connection does not connect.

There is a behaviour difference, the LED now keeps repeatedly retrying whereas previously it was just solid Orange, now it will blink white 6 times and then go Orange again, and then repeat. However no matter how many times it retries I still do not get any internet connectivity.

Log reads as follows :

[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[DHCP IP: (192.168.0.2)] to MAC address 08:84:9D:E4:69:A9                                          
[Internet connected] IP address: (null)                     
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[Internet connected] IP address: (null)                                          
[Internet disconnected]                                          
[DHCP IP: (192.168.0.2)] to MAC address 08:84:9D:E4:69:A9                                          
[Internet connected] IP address: (null)                                          
[admin login] from source 192.168.0.3                                          
[admin login] from source 192.168.0.3                                          
[DHCP IP: (192.168.0.4)] to MAC address 5A:B3:CA:2E:6F:54                                          
[DSL: Up]                                           
[DHCP IP: (192.168.0.13)] to MAC address 02:0F:B5:5F:D1:02                                          
[DHCP IP: (192.168.0.3)] to MAC address A0:E7:0B:FA:27:BF                                          
[DHCP IP: (192.168.0.2)] to MAC address 08:84:9D:E4:69:A9                                          
[Initialized, firmware version: V1.0.1.84_1.0.1 ] 

So it seems to be connecting and then disconnecting.

I have now re-installed wan_orig_0.0.0_armD7000.zip using Package manager over the above installation, rebooted, and regained internet access.

**Comment by Linden1** [comment]
Do you think you can try Installed ppp_2.4.9-001_armD7000.ipk?

**Comment by m0lz** [comment]
> Do you think you can try with ppp_2.4.9-001_armD7000.ipk?

Just tried, installed Build #916, reboot, installed ppp_2.4.9-001_armD7000.ipk, reboot = Thats worse than ppp_2.4.9-002_armD7000.ipk, I get a solid Orange internet LED, log ...

[admin login] from source 192.168.0.3                                          
[admin login] from source 192.168.0.3                                          
[DSL: Up]                                           
[DHCP IP: (192.168.0.13)] to MAC address 02:0F:B5:5F:D1:02                                          
[DHCP IP: (192.168.0.3)] to MAC address A0:E7:0B:FA:27:BF                                          
[DHCP IP: (192.168.0.2)] to MAC address 08:84:9D:E4:69:A9                                          
[Initialized, firmware version: V1.0.1.84_1.0.1 ]

.. At least with ppp_2.4.9-002_armD7000.ipk it keeps trying to connect with the internet as previously posted.

Now reverted to Build #916 with Wan_Orig installed for a working internet.

**Comment by negan07** [comment]
> ppp_2.4.9-002_armD7000
> 
> Working for WAN (ATM) on build 916 grimacing

I wouldn't bet a coin on this but it's good.

ppp-2.4.9-001 is unchanged apart makefiles adaptations for arm-toolchain of course
result:
ATM failure (no chap auth attempt)
PTM failure (no chap auth attempt)

ppp-2.4.9-002 includes broadcom/sercomm/netgear patch adaptations apart pppoe lib plugin
result:
ATM successful
PTM failure (loop on trying to obtain chap auth)

ptm is vdsl on pppoe so it's should be reasonable to obtain an error on it

The original netgear package is a variation of ppp-2.4.1 including (patching) the full libpppoe plugin, in turn patched with broadcom modifications. I need to extract the modifications and to apply them on the 2.4.9 pppoe plugin code because totally different (all the functions & modules are different)

**Comment by negan07** [comment]
Please, it would be useful to know if you have the following variables inside your nvram settings, typing:

```
nvram show | grep ppp_terminate

nvram show | grep pppoe_sid

nvram show | grep pppoe_srv_addr
```

**Comment by Linden1** [comment]
> Please, it would be useful to know if you have the following variables inside your nvram settings, typing:
> 
> ```
> nvram show | grep ppp_terminate
> 
> nvram show | grep pppoe_sid
> 
> nvram show | grep pppoe_srv_addr
> ```

nvram show | grep ppp_terminate
ppp_terminate=1

nvram show | grep pppoe_sid
{no output}

nvram show | grep pppoe_srv_addr
{no output}

**Comment by m0lz** [comment]
I have the same results as Linden1

**Comment by negan07** [comment]
[ppp_2.4.9-003_armD7000.zip](https://github.com/negan07/ancistrus/files/8602873/ppp_2.4.9-003_armD7000.zip)

In this attempt I've include both lib pppoatm & libppoe again with some modifications on makefiles removing new stuffs not used by original vendor code

If not working, a debug on the fly should be done...

those who want to try: take care of `pppd` command line you have on `ps | grep pppd` after trying to connect (it should result as `pppd_1` )

The idea is to kill pppd then re invoking pppd adding the debug flags to see some screen msg if they can help

**Comment by Linden1** [comment]
I have spent 2hrs collecting data for you. I've disconnected so many times I can no longer get a connection, even with build 875.
The following results for ppp_log are for a freshly installed (build) 916 then the various ipk installed.

<b>Bare build 916:</b>
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.

<b>ppp_2.4.9_003_armD7000</b>
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  setting line discipline hook
  Sending PADI
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.
  Clear blacklist.

<b>ppp_2.4.9_002_armD7000</b>
setting line discipline hook
send PPPOE Discovery V1T1 PADI session 0x0 length 12
 dst ff:ff:ff:ff:ff:ff  src 2c:30:33:30:6e:09
 [service-name] [host-uniq  8e 12 00 00]
Recv PPPOE Discovery V1T1 PADO session 0x0 length 49
 dst 2c:30:33:30:6e:09  src 24:21:24:87:32:57
 [service-name] [AC-name LDSCD2-BNG-C1] [host-uniq  8e 12 00 00] [AC-cookie  1b 02 94 39 2b 82 ea d5 cb a0 ea 55 96 83 44 d4]
  Send PPPOE Discovery V1T1 PADR session 0x0 length 32
dst 24:21:24:87:32:57  src 2c:30:33:30:6e:09
 [service-name] [host-uniq  8e 12 00 00] [AC-cookie  1b 02 94 39 2b 82 ea d5 cb a0 ea 55 96 83 44 d4]
Recv PPPOE Discovery V1T1 PADS session 0x73d length 12
 dst 2c:30:33:30:6e:09  src 24:21:24:87:32:57
 [service-name] [host-uniq  8e 12 00 00]
PADS: Service-Name: ''
PPP session is 1853
Connected to 24:21:24:87:32:57 via interface eth0.1
using channel 4
Using interface ppp1
Connect: ppp1 <--> eth0.1
sent [LCP ConfReq id=0x1 <mru 1492> <magic 0xd826d8f4>]
rcvd [LCP ConfReq id=0x5b <mru 1492> <auth chap MD5> <magic 0x43961177>]
sent [LCP ConfAck id=0x5b <mru 1492> <auth chap MD5> <magic 0x43961177>]
rcvd [LCP ConfAck id=0x1 <mru 1492> <magic 0xd826d8f4>]
mtu is <1492> ao->mru is <1492>
Set ppp0 MTU 1492
sent [LCP EchoReq id=0x0 magic=0xd826d8f4]
rcvd [CHAP Challenge id=0x1 <c96bbf5cd4ea1c2164f45436f3df6d9b4081a37512b870ab6c4303ca7a54e9453fa92194>, name = "LDSCD2-BNG-C1"]
sent [CHAP Response id=0x1 <5db14b831f5e12fb12d19f8513da1880>, name = "dsl002295928@broadband.vodafone.co.uk"]
rcvd [LCP EchoRep id=0x0 magic=0x43961177]
rcvd [CHAP Failure id=0x1 "User Object Not Found"]
CHAP authentication failed: User Object Not Found
CHAP authentication failed
sent [LCP TermReq id=0x2 "Failed to authenticate ourselves to peer"]
Script /etc/ppp/ipv6-down started (pid 4766)
rcvd [LCP TermReq id=0x5c]
sent [LCP TermAck id=0x5c]
Script /etc/ppp/ipv6-down finished (pid 4766), status = 0x0
rcvd [LCP TermAck id=0x2]
Connection terminated.


When testing wan_orig it became obvious that my fibre was no longer accepting connections for my D7000, so I won't be much help now until I get that sorted out.
All connections were PPPoE over WAN(ATM)

**Comment by negan07** [comment]
too much incompatibility: I think it's better to regress to ppp-2.4.1

 > I've disconnected so many times I can no longer get a connection, even with build 875

This should be an isp security system due to connection failure trials hammering: it should be solved after a while (leasing time maybe)

**Comment by negan07** [comment]
updated monolithic #917

**Comment by m0lz** [comment]
:) I'll be honest I would have been lost doing the tests Linden1 carried out.

VDSL
Build #917 installed = No problems to report.
Logs seem normal with no errors 👍

**Comment by negan07** [comment]
pppd which is the executable file of ppp-2.4.9, is driven by proprietary binaries like `start_ppp`: they create a copy link of themselves called with the suffix name of the vlan id, usually 1 and then run it. This makes hard to debug it

Killing first `start_ppp` then `pppd_1` then reinvoking pppd_1 with parameters adding debug logfile /tmp/ppp_log

The traditional adsl should work with it, the problem is the pppoe lib for vdsl, totally different and hard to follow without the full code

**Comment by Linden1** [comment]
> Is Linden1 able to connect again hopefully

Currently connected to my isps router. I'll try setting it up again when I get home.

**Comment by negan07** [comment]
try to override the wan mac address setting up another
or the same of the router isp

Lease may require more time, also a mac ban should be temporary

**Comment by Linden1** [comment]
I've tried cloning the mac of another (dead) D7000 and the mac from my isp's router, in combination with build #875 and Netgear's official D7000-V1.0.1.84_1.0.1. For now I think this thing is dead, to be resurrected in a couple of weeks.

**Comment by Linden1** [comment]
I've tried cloning the mac of another (dead) D7000 and the mac from my isp's router, in combination with build #875 and Netgear's official D7000-V1.0.1.84_1.0.1. For now I think this thing is dead, to be resurrected in a couple of weeks.
One thing with build #917. It won't let me connect by connecting my D7000 WAN to my isp router's LAN using dhcp. It's insisting I need to use a username and password. It's constantly trying to connect with ppp even though it's not needed in that mode.

**Comment by negan07** [comment]
Tried yesterday before releasing 917 with ethernet wan mode without issuing. 

`ppp` shouldn't be involved because you are not doing a chap auth in this case until you are using `ipoa` mode
Check the preferences, if the isp allows other vendors routers but it still insist to refuse your connection, ask it for a line reset: sometimes when line conditions drops or many failures happen it may switch you into a failsafe or SOS mode.

**Comment by Linden1** [comment]
> Tried yesterday before releasing 917 with ethernet wan mode without issuing. 
> 
> `ppp` shouldn't be involved because you are not doing a chap auth in this case until you are using `ipoa` mode
> Check the preferences, if the isp allows other vendors routers but it still insist to refuse your connection, ask it for a line reset: sometimes when line conditions drops or many failures happen it may switch you into a failsafe or SOS mode.

I agree ppp should not be involved, but I was definitely being asked for my username and password which were continually rejected.
So I installed build #916 and added wan_orig and set up a dhcp connection using the wizard. Then I updated to build #917 and the settings have stayed and I'm connected through my D7000 through my isp router. My settings survive reboots.
Contacting my isp is the next step. It's not going to be easy though. Vodafone UK customer service is the worst as any customer will attest.

**Comment by m0lz** [comment]
Good luck Linden.
I dont know if this will help but does your ISP have any kind of account settings that typically lock you in to their DNS servers.
I just recall when I got the D7000 to replace the ISP router (objective was so that I could set my own DNS servers for primary and secondary in the router which the ISP router would not allow), I had to go into my ISP account settings and disable a few default setups like BT Parental Control and the ISPs provided anti-malware protection .. And then flush the computer cache and reboot the new router a few times before I could get the internet back. The ISP was unusually quite helpful considering we were discussing a none ISP router which they dont officially support.
Maybe your ISP has defaulted to the new user settings after you were trying to regain access during your testing of the firmware.
I think I am probably teaching you to suck eggs having witnessed the more advanced level of understanding of these things you have already displayed, and this is more than likely well beyond me to try and help with :)

**Comment by Linden1** [comment]
You may very well be right.
Yes, my rapid disconnects will likely have upset my isp, and having ruled out my mac being blacklisted I feel it's highly likely they've changed my login. Unfortunately I can't yet read the login info from a vodafone hub so my only recourse now is to speak to them.

**Comment by negan07** [comment]
> (objective was so that I could set my own DNS servers for primary and secondary in the router which the ISP router would not allow)

this sometimes happen when voip is involved: the router must use internal dns to match voip aliases not reachable from the outside of the isp network

This is not the situation, but there is a trick usable: mantain isp dns on the router then order the dhcp server to configure some different dns as options when a device ask for connection
e.g.
`option dns 1.1.1.1`

on D7000 this must be done manually modifying rc script (there should be an old issue involved)

**Comment by Linden1** [comment]
I'm back online. Vodafone in their infinite wisdom decided to change my login. Why, I don't know. But the recent logs I posted are now redundant.

**Comment by negan07** [comment]
Very good.

The packages are still above on this discussion issue, in any case, if someone has trouble with the originals

**Comment by Linden1** [comment]
Build 917 still has an issue when trying to connect dhcp through wan. ppp is invoked when it shouldn't even be involved with this connection type. Just making you aware because I can see others having this problem. I don't know if it's down to my D7000 router or the vox3 router but I'm not pursuing that route anymore.
wan_orig_0.0.0_ardD7000 is the only fix in this case.

**Comment by negan07** [comment]
> still has an issue when trying to connect dhcp through wan

wan ethernet ?
Are you sure the settings are clean & rebooted ?
Can you post the wan connection page configuration removing mac addresses & logins ?
Can you also post the `ps` result command (with above related deletions) in conjunction with it ?

**Comment by Linden1** [comment]
Yes, WAN Ethernet.
Nothing changes from defaults on the WAN connection page.

**ppp_log** {after login fail}(a step not needed)
setting line discipline hook
Sending PADI
Clear blacklist.
Clear blacklist.
Clear blacklist.
Clear blacklist.
Clear blacklist.

**ps**
PID USER       VSZ STAT COMMAND
    1 root      1248 S    /sbin/init
    2 root         0 SW   [kthreadd]
    3 root         0 SW   [ksoftirqd/0]
    5 root         0 SW   [kworker/u:0]
    6 root         0 SW   [migration/0]
    7 root         0 SW   [migration/1]
    9 root         0 SW   [ksoftirqd/1]
   10 root         0 SW<  [khelper]
   11 root         0 SW   [kworker/u:1]
   26 root         0 SW   [kworker/1:1]
  109 root         0 SW   [sync_supers]
  111 root         0 SW   [bdi-default]
  113 root         0 SW<  [kblockd]
  121 root         0 SW   [khubd]
  130 root         0 SW   [skbFreeTask]
  147 root         0 SWN  [kswapd0]
  148 root         0 SW   [fsnotify_mark]
  151 root         0 SW<  [xfsalloc]
  152 root         0 SW<  [xfs_mru_cache]
  153 root         0 SW<  [xfslogd]
  154 root         0 SW<  [crypto]
  228 root         0 SW   [mtdblock0]
  233 root         0 SW   [mtdblock1]
  238 root         0 SW   [mtdblock2]
  243 root         0 SW   [mtdblock3]
  248 root         0 SW   [mtdblock4]
  253 root         0 SW   [mtdblock5]
  258 root         0 SW   [mtdblock6]
  263 root         0 SW   [mtdblock7]
  268 root         0 SW   [mtdblock8]
  273 root         0 SW   [mtdblock9]
  278 root         0 SW   [mtdblock10]
  283 root         0 SW   [mtdblock11]
  288 root         0 SW   [mtdblock12]
  293 root         0 SW   [mtdblock13]
  298 root         0 SW   [mtdblock14]
  303 root         0 SW   [mtdblock15]
  308 root         0 SW   [mtdblock16]
  313 root         0 SW   [mtdblock17]
  318 root         0 SW   [mtdblock18]
  323 root         0 SW   [mtdblock19]
  328 root         0 SW   [mtdblock20]
  333 root         0 SW   [mtdblock21]
  338 root         0 SW   [mtdblock22]
  343 root         0 SW   [mtdblock23]
  365 root         0 SW<  [linkwatch]
  375 root         0 SW<  [deferwq]
  376 root         0 SW   [kworker/0:1]
  377 root         0 SWN  [jffs2_gcd_mtd0]
  389 root         0 SWN  [jffs2_gcd_mtd4]
  391 root         0 SWN  [jffs2_gcd_mtd3]
  393 root         0 SWN  [jffs2_gcd_mtd8]
  395 root         0 SWN  [jffs2_gcd_mtd12]
  397 root         0 SWN  [jffs2_gcd_mtd11]
  399 root         0 SWN  [jffs2_gcd_mtd10]
  401 root         0 SWN  [jffs2_gcd_mtd2]
  403 root         0 SWN  [jffs2_gcd_mtd16]
  405 root         0 SWN  [jffs2_gcd_mtd14]
  407 root         0 SWN  [jffs2_gcd_mtd17]
  409 root         0 SWN  [jffs2_gcd_mtd9]
  411 root         0 SWN  [jffs2_gcd_mtd15]
  413 root         0 SWN  [jffs2_gcd_mtd18]
  415 root         0 SWN  [jffs2_gcd_mtd19]
  417 root         0 SWN  [jffs2_gcd_mtd20]
  423 root         0 SWN  [jffs2_gcd_mtd21]
  441 root         0 SW   [bcmFlwStatsTask]
  453 root         0 SW   [bcmsw_rx]
  472 root         0 SW   [bcmsw]
  563 root         0 SW   [wl0-kthrd]
  566 root         0 SW   [wfd0-thrd]
  570 root         0 SW   [wl1-kthrd]
  573 root         0 SW   [wfd1-thrd]
  596 root         0 SW   [scsi_eh_0]
  630 root         0 SW   [scsi_eh_1]
  631 root         0 SW   [usb-storage]
  634 root     17492 S    swmdk
  643 root         0 SW   [kworker/1:2]
  649 root       592 S    klogd
  651 root       588 S    cmd_agent_ap
  670 root       596 S    /usr/sbin/rcd
  683 root       604 S    crond
  684 root       776 S    multi_pb_app 90 91 92 10 40
  686 root       592 S    wlevt
  691 root         0 SW   [checkSBusTimeou]
  693 root         0 SW   [NU Work:6945]
  756 root       728 S    /sbin/lan_monitor
  793 root      1248 S    {ad_arpd} /bin/sh /etc/ad_arpd
  816 root      1280 S    /usr/sbin/dbus-daemon --config-file=/etc/dbus-1/syst
  829 root      1668 S    /sbin/syslogd -f /etc/syslog.conf
  846 root         0 SW   [NU TCP]
  848 root         0 SW   [NU UDP]
  852 root     25464 S    KC_BONJOUR
  853 root     25404 S    KC_PRINT
 1009 root      1248 S    /sbin/init
 1427 root       724 S    /usr/sbin/wps_ledctrl
 1714 root         0 SW   [kworker/0:2]
 1870 root       988 S    /usr/sbin/eapd
 1872 root      1420 S    /usr/sbin/nas
 1874 root      1356 S    /bin/wps_monitor
 1877 root       704 S    /usr/sbin/wps_det do_addER_reset
 1934 root      1104 S    /var/dnrd_1 -a 10.0.0.1 -m hosts -c off -r 0 -s 0.0.
 1939 root       600 S    /usr/sbin/utelnetd -l /usr/sbin/login -d
 2020 root         0 SW   [dsl0]
 2029 root      2564 S    /usr/sbin/mini_httpd -d /www -r NETGEAR D7000 -c **.
 2059 root       812 S    /bin/wan_monitor
 2086 root      1044 S    /usr/bin/lld2 group1
 2102 root       892 S    /usr/sbin/miniupnpd -i eth0.1 -a 10.0.0.1 -p 56688 -
 2167 root         0 SW   [ufsd_sda1]
 2205 root      8836 SN   /usr/sbin/smbd -D -P 19
 2208 root      8608 SN   /usr/sbin/nmbd -D -P 19
 2223 root       912 SN   /usr/sbin/bftpd -d -W -P 19
 2279 root       660 S    hd-idle -a sda -i 1800
 2328 root       676 S    /usr/sbin/xCloud_monitor
 2336 root       756 S    /sbin/access_devinfo
 4898 root         0 SW   [flush-mtd-unmap]
 5409 root         0 SW   [kworker/u:2]
 5849 root       692 S    /var/w/start_ppp_1 1
 6416 root       756 S    /var/udhcpd_1 /etc/udhcpd.conf1
 6651 root      1508 S    /var/w/pppd_1 unit 1 ipparam 1 info_file /tmp/w/info
 6777 root      1244 S    sleep 30
 6854 root      1256 S    -sh
 6904 root      1252 R    ps

**Comment by Linden1** [comment]
It's detecting my WAN connection as DSL??

![login1](https://user-images.githubusercontent.com/73253891/166914499-ab2f8d53-c852-4b7b-b58c-5a3c9c9c2cc6.png)

**Comment by Linden1** [comment]
> does this happen also if you setup D7000 wan ip settings manually instead of dhcp ?

Yes. It's like the WAN setup page has no effect or the router doesn't even see the WAN port.

**Comment by negan07** [comment]
> I can see others having this problem
Do these have the same main router interface with same build 917 ?
> wan_orig_0.0.0_armD7000 is the only fix in this case

```
cd /tmp
curl -O https://raw.githubusercontent.com/negan07/ancistrus/gh-pages/tools/debug/pppoe.zip
unzip pppoe.zip
mv -f pppoe /usr/sbin/
curl -O https://raw.githubusercontent.com/negan07/ancistrus/gh-pages/tools/debug/brctl.zip
unzip brctl.zip
mv -f brctl /usr/sbin/
reboot
```

and later on, if nothing good happens

```
cd /tmp
curl -O https://raw.githubusercontent.com/negan07/ancistrus/gh-pages/tools/debug/brctl.zip
unzip brctl.zip
mv -f brctl /usr/sbin/
reboot
```

**Comment by Linden1** [comment]
Each time a fresh install of build 917

pppoe - no change
brctl - no change
pppoe+brctl - no change
pppoe+brctl+ppp_2.4.9-002_armD7000 no change
wan_orig_0.0.0_armD7000 - normal wan dhcp connection.

That's it for tonight. I'll catch up tomorrow.

**Comment by negan07** [comment]
I have included the failsafe `wan_orig_0.0.0` package into the repository.

`opkg update && opkg install wan_orig`

or from web gui `package manager`

Those who have troubles connecting with latest monolithic firmwares, can install it and revert wan connection to original state.

The package can be removed later, because files are installed temporary then moved at postinstall stage script, trick widely used to avoid essential files unwanted deletion.

**Comment by Linden1** [comment]
Problem is if we can't make the WAN connection we can't download it from the repository.

**Comment by negan07** [comment]
You are rigth but with the removal of ppp package the main connection problem issued hopefully should have been fixed.

The wan ethernet configuration has issues related to still undefined particular configurations and hardware involved and can be bypassed switching to switch/ap mode.

To avoid more troubles related, before releasing 917 I tried to recreate & tested a wan ethernet configuration without issuing, but with different hardware involved..

Anyway, bumping a new monolithic with all the regresses is not a problem.
